belgif-rest-problem-it/belgif-rest-problem-spring-boot-3-it/src/test/java/io/github/belgif/rest/problem/RestProblemSpringBoot3IT.java

```diff
@@ -26,10 +25,4 @@ protected Stream<String> getClients() {
         return Arrays.stream(Client.values()).map(Client::name);
     }
 
-    @Override
-    @Disabled("Throws org.springframework.web.servlet.resource.NoResourceFoundException on latest Spring Boot 3")
-    void notFound() {
-        super.notFound();
```

Comment:
This root test only verifies the response has HTTP status 404, it doesn't verify a Problem type with appropriate content was returned.

We should still override it here with additional assertions on the Problem response.